darwin.c

```diff
@@ -7,82 +7,8 @@
 #include <mach-o/dyld.h>
 #include "common.h"
 
-int elevate(char *path, char *prompt, char *iconPath)
```

Comment:
My concern is not to introduce another tool just for this purpose on one OS. But okay, that's just personal opinion.